src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int countOfBalls = 3;
```

Comment:
```suggestion
    private static final int NUMBER_OF_BALLS = 3;
```

https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names